modin/pandas/test/test_io.py

```diff
@@ -885,7 +953,7 @@ def test_read_csv_default_to_pandas(self, request):
         with pytest.warns(UserWarning):
             pd.read_csv(
                 pytest.csvs_names["test_read_csv_regular"],
-                skiprows=lambda x: x in [0, 2],
+                chunksize=10,
```

Comment:
Why are you adding `chunksize` instead of `skiprows` parameter?

Reply:
This test checks default-to-pandas behavior, but since in this PR `read_csv` with `skiprows` not fallbacks to pandas, `skiprows` need to be replaced with parameter, that still fallbacks to pandas implementation (`chunksize` for example).

Reply:
I think it would be more obvious to remove this `with` section.

Reply:
`with` section is needed here to catch warnings produced during pandas implementation call (this test checks that warning is produced).

Reply:
I mean all `with` section with code inside.

Reply:
Oh, i see. Ok, `with` section was removed.